ui/page/settings/view.jsx

```diff
@@ -211,11 +211,20 @@ class SettingsPage extends React.PureComponent<Props, State> {
             <Card
               title={__('Sync')}
               subtitle={
-                walletEncrypted && !storedPassword
-                  ? __("To enable this feature, check 'Save Password' the next time you start the app.")
-                  : null
+                walletEncrypted && !storedPassword ? (
+                  storedPassword === '' ? (
+                    <>
+                      {__('Your wallet password cannot be empty string. Sync is disabled for now.')}
+                      <HelpLink href="https://lbry.com/faq/account-sync" />
+                    </>
+                  ) : (
+                    __(
+                      "Encrypted wallet: To enable Sync, close LBRY completely and check 'Remember Password' during wallet unlock."
```

Comment:
Remove `Encrypted wallet:`

`To enable sync, close the app completely, then check "Remember password" the next time you start it up`